packages/ast-spec/project.json

```diff
@@ -10,7 +10,7 @@
         "cwd": "packages/ast-spec",
         "commands": ["yarn build"]
       },
-      "outputs": ["packages/ast-spec/dist/**/*.ts"]
+      "outputs": ["{projectRoot}/dist/**/*.ts"]
```

Comment:
Unrelated to yarn 3

Reply:
Random thought: one could write Nx-specific lint rules for `project.json` files. There could be a lint rule that suggests replacing hardcoded paths like `"packages/ast-spec` with `"${projectRoot}"`.

Reply:
So this is actually the result of running `nx repair`, as special form of migration that can be run any time to correct any legacy configuration issues or things which are _definitely_ better as far as the tooling can figure out. So I think it's more a case of figuring out a good way to periodically run `nx repair`. I think my existing automated workflow for version migrations makes the most sense for that and I'll add it

Reply:
Added https://github.com/typescript-eslint/typescript-eslint/pull/6162/commits/b549f4e233cc76b21e34368765441b3e41c68337